src/assets/defaultTheme.ts

```diff
@@ -138,6 +138,7 @@ export const defaultTheme: ThemeOptions = {
     },
   },
   palette: {
+    mode: 'dark',
```

Comment:
why should we have an static mode here?

Reply:
We need to define what mode the theme is, if not we will have some misconfiguration with MUI. For now, we just have the _dark_ theme in the library, in the near future we will have two themes, In one it will say dark and light in the other